barretenberg/cpp/src/barretenberg/flavor/ecc_vm.hpp

```diff
@@ -85,715 +87,238 @@ template <typename CycleGroup_T, typename Curve_T, typename PCS_T> class ECCVMBa
+                              precompute_ty,                // column 34
+                              msm_transition,               // column 35
+                              msm_add,                      // column 36
+                              msm_double,                   // column 37
+                              msm_skew,                     // column 38
+                              msm_accumulator_x,            // column 39
+                              msm_accumulator_y,            // column 40
+                              msm_pc,                       // column 41
+                              msm_size_of_msm,              // column 42
+                              msm_count,                    // column 43
+                              msm_round,                    // column 44
+                              msm_add1,                     // column 45
+                              msm_add2,                     // column 46
+                              msm_add3,                     // column 47
+                              msm_add4,                     // column 48
+                              msm_x1,                       // column 49
+                              msm_y1,                       // column 50
+                              msm_x2,                       // column 51
+                              msm_y2,                       // column 52
+                              msm_x3,                       // column 53
+                              msm_y3,                       // column 54
+                              msm_x4,                       // column 55
+                              msm_y4,                       // column 56
+                              msm_collision_x1,             // column 57
+                              msm_collision_x2,             // column 58
+                              msm_collision_x3,             // column 59
+                              msm_collision_x4,             // column 60
+                              msm_lambda1,                  // column 61
+                              msm_lambda2,                  // column 62
+                              msm_lambda3,                  // column 63
+                              msm_lambda4,                  // column 64
+                              msm_slice1,                   // column 65
+                              msm_slice2,                   // column 66
+                              msm_slice3,                   // column 67
+                              msm_slice4,                   // column 68
+                              transcript_accumulator_empty, // column 69
+                              transcript_reset_accumulator, // column 70
+                              precompute_select,            // column 71
+                              lookup_read_counts_0,         // column 72
+                              lookup_read_counts_1);        // column 73
```

Comment:
75 members before = 73 members now + 2 derived witnesses `z_perm` and `lookup_inverse` ✅